skins/oasis/modules/templates/CommunityPageEntryPointController_Index.php

```diff
@@ -1,5 +1,15 @@
-<section class="community-page-entry-point-module module">
-	<div class="community-page-entry-point-logo"></div>
-	<div class="community-page-entry-point-description"><?= wfMessage( 'communitypage-help-us-grow' )->parse() ?></div>
-	<a href="<?= SpecialPage::getTitleFor( 'Community' )->getLocalURL(); ?>" class="community-page-entry-point-button"><?= wfMessage( 'communitypage-entry-button' )->escaped() ?></a>
+<section class="community-page-entry-point-module">
+	<div class="avatars">
+		<? foreach( $avatars as $index => $avatar ): ?>
+			<a href="<?= $avatar['userProfileUrl'] ?>">
+				<img class="wds-avatar" style="left:<?= ($index * -10) + ( count( $avatars ) - 1 ) * 5 ?>px; z-index: <?= 5 - $index ?>" src="<?= $avatar['avatarUrl'] ?>" />
```

Comment:
maybe we could move these inline styles to scss or use desing system https://github.com/Wikia/design-system/blob/01699b6b06330d144c311ffd28cef332910a4ad5/style-guide/styles/wds-components/_avatar-stack.scss#L10-L10?